recipe-server/client/control/routes.js

```diff
@@ -42,30 +42,35 @@ const routes = {
           '/': {
             component: RecipeDetailPage,
             crumb: 'Revision',
+            ignoreSession: true,
```

Comment:
Can you explain the choice of name `ignoreSession`? I don't think I get it.

It might also be good to have a comment above the routes explaining the extra keys we are adding. They all make sense now, but they aren't documented anywhere, and the schema is pretty important.

Reply:
The idea behind `ignoreSession` was to ignore the route for session purposes, but I agree it is/was too vague. I've switched over to this format:

```
...
'/clone': {
  '/': {
    component: CloneRecipePage,
    crumb: 'Clone Recipe',
    sessionSlug: 'recipe-view',
    slug: 'recipe-clone',
  },
},
...
```

With this approach, we can simply point to another slug and anything session-related will use that over the actual `slug`. This feels a bit better since the `routes.js` file now lets us define the redirect behavior without having to tie directly into the underlying breadcrumb/current URL stuff.